### PR TITLE
PLAT-20711 doc: add support center link

### PR DESCRIPTION
**Pull request by DemonTsai**

### CODE REVIEW

File: source/futures/cn/index.html.md [hunk 1]
```diff
@@ -490,11 +496,12 @@ BTSE 的速率限制如下：
...
 
-* 订单等于或低于5个合约将被标记为垃圾订单，并自动变为隐藏订单。
+[垃圾订单检测机制 : BTSE Support](https://support.btse.com/en/support/solutions/articles/43000720904-spam-order-detection-mechanism)
+
+* 订单的名义价值低于5美元的将被标记为垃圾订单，并自动变为隐藏订单。
 * 被标记为垃圾的订单始终支付吃单费。
 * 被标记为垃圾的Post-Only API订单将被拒绝而不是被隐藏。
 * 太多的垃圾订单可能导致暂时封禁交易账户。
-* 放置 >= 4 个挂单，总大小小于 20 个合约的API账户有可能被标记为垃圾账户。
 * 被标记为垃圾的账户可能会对账户施加限制，包括订单速率限制、持仓限制，或禁用API功能。如对新的垃圾订单机制有疑问，请发送电子邮件至 mm@btse.com。
```
Comment (jerryLi850) [inline source/futures/cn/index.html.md:507]: Please remove this blank line as well to sync the change with en version.

File: source/futures/cn/index.html.md [hunk 2]
```diff
@@ -13,6 +13,12 @@ headingLevel: 2
 
 # 更新日志
 
+## 版本 2.7.3 (2024年2月19日)
+
+* 添加垃圾订单检测机制：BTSE支持链接
```
Comment (jerryLi850) [inline source/futures/cn/index.html.md:18]: Why does the English version have hyperlinks while the Chinese version does not?

Comment (DemonTsai) [reply source/futures/cn/index.html.md]: done.

File: source/futures/cn/index.html.md (hunk 1 above)
Comment (DemonTsai) [reply source/futures/cn/index.html.md:507]: 剛得知消息這一條不用移除了

Comment (jerryLi850) [reply source/futures/cn/index.html.md:507]: 其實我說的是空白行ＸＤ

Comment (DemonTsai) [reply source/futures/cn/index.html.md]: done

**Review by jerryLi850 — APPROVED** [review]

**Pull request merged**